src/menu/MenuSongSelection.as

```diff
@@ -603,425 +631,976 @@ package menu
-                    ratingDisplay.value = songDetails['song_rating'];
-                    ratingDisplay.rotation = 90;
-                    ratingDisplay.scaleX = ratingDisplay.scaleY = 0.60;
-                    info.addChild(ratingDisplay);
+                    var songHighscoresButton:BoxButton = new BoxButton(79.5, 27, (options.infoTab == TAB_HIGHSCORES ? _lang.string("song_selection_song_panel_info") : _lang.string("song_selection_song_panel_scores")), 12);
+                    songHighscoresButton.x = 89.5;
+                    songHighscoresButton.y = 256;
+                    songHighscoresButton.level = songDetails.level;
+                    songHighscoresButton.action = "highscores";
+                    songHighscoresButton.addEventListener(MouseEvent.CLICK, clickHandler, false, 0, true);
+                    infoBox.addChild(songHighscoresButton);
                 }
-                for (var item:String in infoDisplay)
+
+                var songStartWidth:Number = 164;
+                if (_mp.gameplayCanPick())
                 {
-                    // Info Title
-                    songInfoTitle = new Text(infoDisplay[item][0], 14, "#DDDDDD");
-                    songInfoTitle.x = 0;
-                    songInfoTitle.y = tY;
-                    songInfoTitle.width = 164;
-                    info.addChild(songInfoTitle);
-                    tY += 16;
-
-                    // Info Display
-                    songInfoDetails = new Text(infoDisplay[item][1]);
-                    songInfoDetails.x = 0;
-                    songInfoDetails.y = tY;
-                    songInfoDetails.width = 164;
-                    info.addChild(songInfoDetails);
-                    tY += 23;
+                    songStartWidth = 79.5;
+                    var songLoadButton:BoxButton = new BoxButton(79.5, 27, _lang.string("song_selection_song_panel_mp_load"), 14);
+                    songLoadButton.x = 89.5;
+                    songLoadButton.y = 288;
+                    songLoadButton.level = songDetails.level;
+                    songLoadButton.addEventListener(MouseEvent.CLICK, songLoadClick, false, 0, true);
+                    infoBox.addChild(songLoadButton);
                 }
+                var songStartButton:BoxButton = new BoxButton(songStartWidth, 27, _lang.string("song_selection_song_panel_play"), 14);
+                songStartButton.x = 5;
+                songStartButton.y = 288;
+                songStartButton.level = songDetails.level;
+                songStartButton.addEventListener(MouseEvent.CLICK, songStartClick, false, 0, true);
+                infoBox.addChild(songStartButton);
+            }
+            else
+            {
+                var songHighscoresButtonLocked:BoxButton = new BoxButton(164, 27, (options.infoTab == TAB_HIGHSCORES ? _lang.string("song_selection_song_panel_info") : _lang.string("song_selection_song_panel_scores")), 12);
+                songHighscoresButtonLocked.y = 288;
```

Comment:
```suggestion
                songHighscoresButtonLocked.x = 5;
                songHighscoresButtonLocked.y = 288;
```
I think we should add this line so that the button is aligned with the center of the info box.

Reply:
Fixed.